lib/golomb/README.md

```diff
@@ -0,0 +1,26 @@
+# Golomb
+
+Golomb `lib/golomb` is a bcoin module intended to be used as a Golomb Coded Set
+for creating Compact Block Filters as specified in [BIP 158][0].  It is used to
+create block filters which can be used to quickly scan blocks for a given set
+of transaction outputs.
+
+Block filters are stored as an index using the blockstore.  To enable indexing
+filters, the config option `index-filter` should be enabled.  For mainnet, the
+indexing requires several hours on average and occupies ~4GB disk space.
+
+Block filters can be accessed using the RPC:
+
+`getblockfilter <hash>`
+
+or the HTTP API:
+
+`127.0.0.1:18556/filter/<height|hash>`
```

Comment:
I think something like `GET /filter/<height|hash>` would be more useful than providing a hostname and port because they can vary based on the network or config and this implies that filters will always be served at that address.

Reply:
:+1: